knex/index.d.ts

```diff
@@ -251,10 +259,12 @@ declare namespace Knex {
         (columnName: string, direction?: string): QueryBuilder;
     }
 
+    type UnionFunction = (this: QueryBuilder) => void;
+
     interface Union {
-        (callback: Function, wrap?: boolean): QueryBuilder;
-        (callbacks: Function[], wrap?: boolean): QueryBuilder;
-        (...callbacks: Function[]): QueryBuilder;
+        (callback: UnionFunction | UnionFunction[], wrap?: boolean): QueryBuilder;
+        (...callbacks: UnionFunction[]): QueryBuilder;
+        // no way to specifiy this siganiture in the current typescript version
```

Comment:
Got it, thanks. Could you fix the typos in the comment and clarify that it refers to the  line below?